systems/registry/org/org_reg.patch

```diff
@@ -0,0 +1,2424 @@
+diff --git a/systems/registry/org/cmd/server/main.go b/systems/registry/org/cmd/server/main.go
```

Comment:
this file should be removed  and checked out of git repo  (git rm)

Reply:
Addressed